pool/store/sqlite_store.py

```diff
@@ -167,7 +167,12 @@ async def clear_farmer_points(self) -> None:
         cursor = await self.connection.execute(f"UPDATE farmer set points=0")
         await cursor.close()
         await self.connection.commit()
-
+        
+    async def auto_delete_partial(self,retetion_time: uint64) -> None:
+        cursor = await self.connection.execute(f"DELETE from partial where timestamp <?", (retetion_time,))
```

Comment:
The timestamp on the table is a UNIX time,. so this is not correct

Reply:
await self.store.auto_delete_partial(time.time() - self.partial_retetion_time)

retetion_time = time.time() - pool_config["partial_retetion_time"]

DELETE from partial where timestamp < retetion_time 
It means delete the data before (current time - pool_config["partial_retetion_time"]).